### DISCUSSION TITLE
Use NVML except Xnvctrl in Wayland

### DISCUSSION THREAD

**Discussion by QkiZMR** [brndnmtthws/conky#2121] [Ideas]
Recently, I switched to Wayland. The Nvidia graphs (gfx load, memory usage) that I have in Conky config stopped working. I had a similar problem with MangoHud on Wayland, but I needed to compile MangoHud with the NVML option enabled, and now it detects Nvidia gfx properly. 

The question is: can Conky use the same NVML mechanism to get metrics from the NVidia card?

**Comment by Caellian** [comment]
Let's keep the discussion in #1615, I'll reopen it as it's older and has more context.
